README.md

```diff
@@ -475,8 +475,8 @@ which you may make direct use of as follows:
 import com.databricks.spark.xml.XmlInputFormat
 
 // This will detect the tags including attributes
-sc.hadoopConfiguration.set(XmlInputFormat.START_TAG_KEY, "<books>")
-sc.hadoopConfiguration.set(XmlInputFormat.END_TAG_KEY, "</books>")
+sc.hadoopConfiguration.set(XmlInputFormat.START_TAG_KEY, "<book>")
+sc.hadoopConfiguration.set(XmlInputFormat.END_TAG_KEY, "</book>")
 sc.hadoopConfiguration.set(XmlInputFormat.ENCODING_KEY, "utf-8")
 
 val records = context.newAPIHadoopFile(
```

Comment:
Oh, wait, do you mind just to change `val records = context.newAPIHadoopFile(` to `val records = sc.newAPIHadoopFile(`? It'd be great if typos are fixed together in this example!